api/src/main/java/org/pmiops/workbench/db/dao/projection/PrjWorkspace.java

```diff
@@ -0,0 +1,75 @@
+package org.pmiops.workbench.db.dao.projection;
+
+import java.sql.Timestamp;
+import org.pmiops.workbench.model.BillingAccountType;
+import org.pmiops.workbench.model.BillingStatus;
+
+public interface PrjWorkspace {
+  BillingAccountType getBillingAccountType();
```

Comment:
Some of the enum columns "just work", but others (like `data_access_level`) don't. I believe it had to do with which attribute on the entity class was annotated with the column name.

I wound up leaving DataAccessLevel out as it was one of those stubborn columns and the representation is slated to change in the multiple tiers project.